tests/MentionsInput.spec.tsx

```
@@ -4651,6 +4659,60 @@ describe('MentionsInput', () => {
+      const { unmount } = render(
+        <MentionsInput ref={ref} value="@a" suggestionsDisplay="inline">
+          <Mention trigger="@" data={data} />
+        </MentionsInput>
+      )
+
+      const instance = ref.current as unknown as any
+      const control = document.createElement('div')
+      const highlighter = document.createElement('div')
+      const caret = document.createElement('span')
+
+      caret.setAttribute('data-mentions-caret', 'true')
```

ESLint

[error] Prefer `.dataset` over `setAttribute(…)`.

(unicorn/prefer-dom-node-dataset)